src/content/docs/es/guides/integrations-guide/cloudflare.mdx

```diff
@@ -201,6 +287,49 @@ export default defineConfig({
 });
 ```
 
+## Importaciones de módulos Wasm
+
+`wasmModuleImports: boolean`
+
+por defecto: `false`
+
+Si importar o no los archivos `.wasm` [directamente como módulos ES](https://github.com/WebAssembly/esm-integration/tree/main/proposals/esm-integration).
+
+Agrega `wasmModuleImports: true` a `astro.config.mjs` para habilitar tanto en la compilación de Cloudflare como en el servidor de desarrollo de Astro.
+
+```diff
+// astro.config.mjs
+import {defineConfig} from "astro/config";
+import cloudflare from '@astrojs/cloudflare';
+
+export default defineConfig({
+    adapter: cloudflare({
++       wasmModuleImports: true
+    }),
+	  output: 'server'
+})
+```
+
+Una vez habilitado, puedes importar un módulo de ensamblaje web en Astro con una importación `.wasm?module`.
```

Comment:
```suggestion
Una vez habilitado, puedes importar un módulo de ensamblador web en Astro utilizando una importación con la extensión `.wasm?module`.
```

Reply:
I don't know if we should translate the Web Assembly. It is a concept that does not need to be updated.

Reply:
¿Qué opinas @Waxer59?

Reply:
Basándome en otras documentaciónes como `mdn` opino que si podemos usar `web assembly` sin problema alguno